### ISSUE TITLE
Whitenoise:2.0 fails to recognize function/subroutine calls wrapped in if statements.

### ISSUE THREAD

**Issue by feckertson** [cerner/ccl-testing#9]

**Comment by feckertson** [comment]
var0 is inappropriately flagged as an unused variable because the call to sub0 is not recognized.
```drop program prog go 
create program prog
    declare var0 = i4
    subroutine(sub0(null) = i2)
        call echo(var0)
        return (TRUE)
    end
    if (sub0(null))
        call echo('sub0 success')
    endif
end go
```

**Comment by feckertson** [comment]
Fixed with #11.
